tests/puppeteer/pages/BO/customers/addresses/index.js

```diff
@@ -15,14 +15,15 @@ module.exports = class Addresses extends BOBasePage {
     this.addressGridPanel = '#address_grid_panel';
     this.addressGridTitle = `${this.addressGridPanel} h3.card-header-title`;
     this.addressesListForm = '#address_grid';
-    this.addressesListTableRow = `${this.addressesListForm} tbody tr:nth-child(%ROW)`;
-    this.addressesListTableColumn = `${this.addressesListTableRow} td.column-%COLUMN`;
-    this.addressesListTableColumnAction = this.addressesListTableColumn.replace('%COLUMN', 'actions');
-    this.addressesListTableToggleDropDown = `${this.addressesListTableColumnAction} a[data-toggle='dropdown']`;
-    this.addressesListTableDeleteLink = `${this.addressesListTableColumnAction} a[data-url]`;
-    this.addressesListTableEditLink = `${this.addressesListTableColumnAction} a[href*='edit']`;
+    this.addressesListTableRow = row => `${this.addressesListForm} tbody tr:nth-child(${row})`;
+    this.addressesListTableColumn = (row, column) => `${this.addressesListTableRow(row)} td.column-${column}`;
+    this.addressesListTableColumnAction = row => this.addressesListTableColumn(row, 'actions');
+    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)
+    } a[data-toggle='dropdown']`;
```

Comment:
```suggestion
    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)} a[data-toggle='dropdown']`;
```

Reply:
I have max length eslint error with this suggestion.

Reply:
```suggestion
    this.addressesListTableToggleDropDown = row => `${this.addressesListTableColumnAction(row)} `
    + ` a[data-toggle='dropdown']`;
```
for al lines

Reply:
done thanks